### PR TITLE
allow the user choose the namespace of the redis

### PR DESCRIPTION
**Pull request by DiogoMarques2003**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/rate-limit",
-  "version": "7.1.0",
+  "version": "7.1.1",
```
Comment (mcollina) [inline package.json:3]: Please revert this, we'll bump when releasing.

**Review by mcollina — CHANGES_REQUESTED** [review]

**Review by mcollina — CHANGES_REQUESTED** [review]
Thanks for opening a PR! Can you please add a unit test?

**Comment by DiogoMarques2003** [comment]
> Thanks for opening a PR! Can you please add a unit test?

Unit teste for what ?

**Comment by Eomm** [comment]
> Unit teste for what ?

That the redis key is what you expext

> I don't think it is necessary to add any more unit tests since it was only added an optional parameter in the options

In this way, we can't assure that this feature will be removed in future.

**Comment by DiogoMarques2003** [comment]
@Eomm is this ? Can you check pls

**Review by Eomm — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**